client/src/components/CredentialsButton/CredentialsButton.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./CredentialsButton.css";
+import { PropTypes } from "prop-types";
+
+export const CredentialsButton = ({ data, onClick }) => {
+  return (
+    <button className="credentials" onClick={onClick}>
```

Comment:
The onClick function could have a rather better name as well. Something that's more descriptive that can provide a bit of insight as to what the onClick function would perform

Reply:
Fixed: changed to `onClick` to `authAction`.